### ISSUE TITLE
Start using Cake.Recipe for windows builds

### ISSUE THREAD

**Issue by AdmiringWorm** [cake-contrib/Cake.Transifex#9]

**Comment by gep13** [comment]
@AdmiringWorm out of interest, is there something on the Travis build that still isn't working when using Cake.Recipe?

**Comment by AdmiringWorm** [comment]
Last time I checked it, there was a compile error when building the script.
Can't remember what though unfortunately, as well as I've been unable to build everything correctly when targeting both .NET Framework and .NET Core.

**Comment by gep13** [comment]
>@AdmiringWorm said...
as well as I've been unable to build everything correctly when targeting both .NET Framework and .NET Core.

Can you elaborate on this?  Perhaps raise an issue on Cake.Recipe repo? Thank you!

**Comment by AdmiringWorm** [comment]
Basically, it' because of me targeting .NET Framework, and the .NET Core utility are unable to find the Mono library directory (this is a known bug in .NET Core).

There is actually a workaround for it, to export the (environment?) variable `FrameworkPathOverride` to the directory where the necessary libraries are located.
I managed to get that working for Linux, but not for MacOS (and I don't have any experience with OSX).

That's why I added the note in the Building documentation for linux:
https://cake-contrib.github.io/Cake.Transifex/docs/building/linux
